### PR TITLE
✨ feat: Improvements about synchronising requisitions and tagging

### PR DESCRIPTION
**Pull request by indigo423**
Instead of running tasks in the playbook, I've added a conditional to run the import requisition node with the last node in the play for all hosts in a group. The requisition import task is also tagged and can be skipped with `--extra-vars '{"skip_import":"true"}'`

Resolves: #60

### CODE REVIEW

**Comment by indigo423** [comment]
@mfuhrmann It would be nice if you can give the requisition import behavior some testing with multiple nodes in at least two groups that end up in requisitions.

**Comment by indigo423** [comment]
@mfuhrmann if you can give it a spin and merge it would be nice. I'm afk for a while and can't do anything about it :)

**Comment by mfuhrmann** [comment]
I found one small issue with the handler name that I fixed within this PR otherwise the playbook didn't work.
Skipping the import worked for me :+1: I mentioned this feature in the README.

**Review by mfuhrmann — APPROVED** [review]

File: ansible/roles/horizon-provision/tasks/main.yml [hunk 1]
```diff
@@ -97,3 +109,24 @@
   register: service_created
   when: service_exists.failed
   with_dict: "{{ onms_group_services | combine(onms_host_services) }}"
+  tags:
+    - opennms-provisioning
+
+# ansible_play-hosts:
+# List of hosts in the current play run, not limited by the serial. Failed/Unreachable hosts are excluded from this list.
+# With the condition inventory_hostname == ansible_play_hosts[-1] we run the task only for the last host in the current play
```
Comment (mfuhrmann) [inline ansible/roles/horizon-provision/tasks/main.yml:117]: @indigo423 This was the old way of doing it, right?

**Comment by mfuhrmann** [comment]
With the latest commit we will always run the import only at the end (last entry in host inventory). But the condition in the task will allow to skip the import.

**Pull request merged**